packages/toolkit/src/amazonqFeatureDev/session/session.ts

```diff
@@ -51,12 +51,6 @@ export class Session {
             await this.setupConversation(msg)
             this.preloaderFinished = true
 
-            telemetry.amazonq_startChat.emit({
```

Comment:
Were we explicitly asked to remove this? The Q team might be using this metric in their dashboards.

Reply:
Yes. See above. Was asked to remove it.